### PR TITLE
controllers with basic functionality

### PR DESCRIPTION
**Pull request by matua**

### CODE REVIEW

File: src/main/java/com/matuageorge/technicalmaintenanceschedule/controller/TaskController.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+public class TaskController {
+
+    private final TaskService taskService;
+
+    @PostMapping("/admin")
+    public ResponseEntity<Task> save(@RequestBody TaskDto taskDto) throws ValidationException, ResourceAlreadyExistsException {
+
+        log.info("Handling task {} to create", taskDto);
+
+        Task savedTask = taskService.save(taskDto);
```
Comment (caligula95) [inline src/main/java/com/matuageorge/technicalmaintenanceschedule/controller/TaskController.java]: you can inline this

Comment (caligula95) [reply src/main/java/com/matuageorge/technicalmaintenanceschedule/controller/TaskController.java]: please read article about naming conventions in rest. and about rest principles.https://blog.dreamfactory.com/best-practices-for-naming-rest-api-endpoints/
Endpoint tasks/admin say nothing about what is going on.

File: src/main/java/com/matuageorge/technicalmaintenanceschedule/controller/TerminalController.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    @PostMapping("/admin")
+    public ResponseEntity<Terminal> save(@RequestBody TerminalDto terminalDto) throws ValidationException, ResourceAlreadyExistsException {
+
+        log.info("Handling terminal {} to create", terminalDto);
+
+        Terminal savedTerminal = terminalService.save(terminalDto);
+        return ResponseEntity.ok(savedTerminal);
+    }
+
+    @PutMapping("/admin")
```
Comment (caligula95) [inline src/main/java/com/matuageorge/technicalmaintenanceschedule/controller/TerminalController.java]: you can put /admin to global variables and declare it only once.

File: src/main/java/com/matuageorge/technicalmaintenanceschedule/controller/TaskController.java (hunk 1 above)
Comment (matua) [reply src/main/java/com/matuageorge/technicalmaintenanceschedule/controller/TaskController.java]: ok

**Pull request merged**